Geometry/RPCGeometryBuilder/src/RPCGeometryBuilderFromDDD.cc

```diff
@@ -25,14 +32,21 @@
 
```

Comment:
please, remove CLHEP and use only CMSUnits

Reply:
@ianna Done!